.travis/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+
+name=$1
+health=$2
+options=$3
+
+docker run --detach --name=${name} --health-cmd="${health}" ${options} > /dev/null
+
+while STATUS=$(docker inspect --format "{{.State.Health.Status }}" ${name}); [[ ${STATUS} != "healthy" ]]; do
+    if [[ ${STATUS} == "unhealthy" ]]; then
+        printf "%s is unhealthy\n" "$name"
+        exit 1
+    fi
+    printf .
+    sleep 1
+done
+printf "\n%s is healthy\n" "$name"
```

Comment:
I always run `shellcheck` on these scripts:
```
In .travis/run.sh line 8:
docker run --detach --name=${name} --health-cmd="${health}" ${options} > /dev/null
                           ^-- SC2086: Double quote to prevent globbing and word splitting.
                                                            ^-- SC2086: Double quote to prevent globbing and word splitting.


In .travis/run.sh line 10:
while STATUS=$(docker inspect --format "{{.State.Health.Status }}" ${name}); [[ ${STATUS} != "healthy" ]]; do
                                                                   ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
Though not sure if quotes can be used on `${options}`